integration-tests/bindy/src/main/java/org/apache/camel/quarkus/component/bindy/it/BindyResource.java

```diff
@@ -110,6 +111,19 @@ public void unMarshalFixedLengthRecordShouldSucceed() {
         assertEquals("Spa", order.getCountry());
     }
 
+    @Path("/marshalFixedLengthWithLocaleShouldSucceed")
+    @GET
+    public void marshalFixedLengthWithLocaleShouldSucceed() {
+        LOG.debugf("Invoking marshalFixedLengthWithLocaleShouldSucceed()");
+
+        FixedLengthWithLocale object = new FixedLengthWithLocale();
+        object.setNumber(3.2);
+
+        String marshalled = template.requestBody("direct:marshal-fixed-length-with-locale", object, String.class);
+
+        assertEquals("٣٫٢٠٠\r\n", marshalled);
```

Comment:
nice!

Reply:
Yeah, this is worldwide camel-quarkus :)